### PR TITLE
feat(github issues image): add image w/ scripts

### PR DESCRIPTION
**Pull request by kirederik**
related to https://github.com/syntasso/kratix-marketplace/issues/9

system-testing this one is a bit annoying (as it requires a bit of a setup); leaving it without tests for now

### CODE REVIEW

**Comment by kirederik** [comment]
@abangser could you take a look at this one when you have the time? 👀 once merged i can add it to the marketplace

**Review by abangser — COMMENTED** [review]
Sorry @kirederik, in clearing through my notifications I noticed I never hit `submit review` on this. Nothing in here is blocking anyways, but would be good to get the typos fixed and to understand the thinking about status vs status AND metadata when you get a chance!

File: pipeline-marketplace-images/github-sign-off/README.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+kubectl create secret generic github-token --from-literal=token=<YOUR GITHUB TOKEN>
+```
+
+Please follow the [Github
+documentation](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
+for details on how to generate the token.
+
+## Usage
+
+Usage of this image, you must add two workflows to your Promise.
```
[the remark this answers is not in the document]
Comment (kirederik) [reply pipeline-marketplace-images/github-sign-off/README.md:21]: yeah the two workflows are there to bubble up the message to the user that the request is pending approval. You could do it as a single workflow, but then the requester would be just confused on why the pipeline is not running.

i will add a note with the alternative

File: pipeline-marketplace-images/github-sign-off/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# Github Issues Gate
```
[the remark this answers is not in the document]
Comment (kirederik) [reply pipeline-marketplace-images/github-sign-off/README.md:1]: yeah right now it will just auto-approve; i added some docs

**Pull request merged**